.github/workflows/docker.yml

```diff
@@ -0,0 +1,49 @@
+name: Docker Build
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        default: "dev"
+        required: false
+        type: string
```

Comment:
Could you please clarify how you are going to use it ?

Reply:
We can run it from actions tab, to build it manually, useful when running in SCT with `:dev` tag for testing without making a TAG

Reply:
Then it should be `workflow_dispatch` not `workflow_call`

Reply:
You dont have to do anything in the code, as that is by default provided in GitHub, you can choose branch